src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Data/TestDataHelper.cs

```diff
@@ -140,7 +140,7 @@ public static DelegationChange GetDelegationChange(string altinnAppId, int offer
                 Created = DateTime.Now,
                 IsDeleted = false,
                 OfferedByPartyId = offeredByPartyId,
-                PerformedByUserId = 20001336,
+                PerformedByUserId = performedByUserId.GetValueOrDefault(),
```

Comment:
This is test/mocking code so we can allow for some creativity, but the use of `GetValueOrDefault()` is curious. I assume it is being used because `PerformedByUserId` is a regular int and not nullable (int?).

Just understand that if somone calls the `GetDelegationChange` method with `performedByUserId = null` the `PerformedByUserId` property will get the value 0.

Reply:
Reverted these changes. My tests automatically used a different overloaded `GetDelegationChange` and ran ok.